### DISCUSSION TITLE
Unable to get PIP to float overtop of fullscreen window

### DISCUSSION THREAD

**Discussion by i-like-buff-green-women** [i3/i3#5984] [Q&A]
I'm new to Arch and I3, so please forgive my greenness.

I frequently use Firefox's PIP feature to watch videos while I have a fullscreen app open, usually a game.  I haven't been able to get this to work on I3.

I've been researching and troubleshooting for at least an hour now.  I've tried using SUPER+SPACE, as someone suggested, but this didn't work, it just kicked me out of full screen and shifted focus between the app I was using and the PIP window.  This was the most relevant thing I've come across so far.

Disclaimers to save someone time if they have these suggestions:
-I am not interested in using a 2nd monitor, I don't have the space.
-I do not want to have to custom resize windows, that defeats the purpose of the PIP function.  I've seen this recommended and want to explore options to have the native PIP features work instead.

Any info anyone has would be appreciated.  Thanks in advance.

**Comment by awsms** [comment]
Same here :(

**Comment by orestisfl** [comment]
This is by design.

However, the new `popup_during_fullscreen all` option will display the PIP window over all fullscreen windows. You will need to either wait for the 4.24 release or use i3 from the `next` branch.
